simtools/tests/test_input_validation.py

```diff
@@ -50,5 +50,57 @@ def test__validate_data_type_email():
         match=r"invalid email format in field EMAIL: me-blabla.de"):
             date_validator._validate_data_type(email_schema1, email_key, 'me-blabla.de')
 
+def test__validate_data_type_othertypes():
+
+    date_validator = validator.SchemaValidator(None, None)
+    test_key = 'SUBTYPE'
+    test_schema_1 = {'type': 'str'}
```

Comment:
Could these be changed to be `test_schema_str`, `test_schema_float`, etc.? Would be much easier to read.

Reply:
Yes! Done.